app/assets/javascripts/application.js

```diff
@@ -37,3 +28,18 @@ function stripeResponseHandler(status, response) {
     $form.get(0).submit();
   }
 };
+
+$(document).ready(function() {
+  $("#use_saved_card").on("click", toggleCreditCardFields);
+});
+
+function toggleCreditCardFields(){
```

Comment:
Purely a stylistic concern -- I'd include a space character after the parens for the parameters and before the left curly brace whenever declaring functions, so like `toggleCreditCardFields() {`